README.md

```diff
@@ -13,7 +13,7 @@ SafetyHook is a procedure hooking library for Windows x86 and x86_64 systems. It
 
 ## Installation
 
-SafetyHook can be added via CMake's `FetchContent`, git submodules, or copied directly into your project. By default it comes bundled with the [Zydis](https://github.com/zyantific/zydis) disassembler. If you already have [Zydis](https://github.com/zyantific/zydis) integrated into your project you can disable SafetyHook's bundling by using the CMake option `-DSAFETYHOOK_BUNDLE_ZYDIS=OFF`.
+SafetyHook can be added via CMake's `FetchContent`, git submodules, or copied directly into your project. By default, it comes bundled with the [Zydis](https://github.com/zyantific/zydis) disassembler. If you already have [Zydis](https://github.com/zyantific/zydis) integrated into your project you can disable SafetyHook's bundling by using the CMake option `-DSAFETYHOOK_BUNDLE_ZYDIS=OFF`.
```

Comment:
Maybe change this since `SAFETYHOOK_BUNDLE_ZYDIS` is no longer an option. Zydis is now expected to already be present, or is bundled in the amalgamated build.

Reply:
Or can be fetched with `SAFETYHOOK_FETCH_ZYDIS` or w/e i called it :)

Reply:
I think this entire section could use a rewrite. I'll try it out.